docs/package.json

```diff
@@ -79,5 +79,15 @@
         "sucrase": "^3.32.0",
         "typescript": "^4.7.4",
         "webpack": "^5.88.1"
+    },
+    "pnpm": {
+        "packageExtensions": {
+            "react-simple-code-editor": {
+                "peerDependencies": {
+                    "react": "^17.0.2",
+                    "react-dom": "^17.0.2"
+                }
+            }
+        }
```

Comment:
@johnnyomair Is this okay? Do I have to do it in the root package.json?

Since the error doesn't occur locally, I couldn't test it